olca-app/src/org/openlca/app/M.java

```diff
@@ -1019,6 +934,14 @@ public class M extends NLS {
 	private M() {
 	}
 
+	public static String bindParenthesis(String primary, String secondary) {
+		return String.format("%s (%s)", primary, secondary);
+	}

```

Comment:
@msrocka In the same vein as [this comment]([url](https://github.com/GreenDelta/olca-app/pull/392#discussion_r1611121723)). You could also share your thoughts on this.

Reply:
I would also not do this. Just put these kind of messages with their parentheses in message files; the messages can change, using logic to construct them makes it harder to maintain them than simply putting them with nicely labeled keys into message files